### PR TITLE
[playground] Updates canUseApps condition to also consider counterparty's FreeBalance state

### PR DESCRIPTION
**Pull request by joelalejandro**
This PR removes the multisig balance check from the `canUseApps` condition to use the counterparty's FreeBalance as part of the evaluation.

Also, it restricts the minimum viable deposit to `0.01 ETH`.

### CODE REVIEW

File: packages/playground/src/components/app-root/app-root.tsx [hunk 1]
```diff
@@ -255,26 +257,50 @@ export class AppRoot {
...
+        ethers.utils.HDNode.fromExtendedKey(xkey).derivePath(String(k))
+          .publicKey
+      );
+
+    const balances = [
+      ethers.utils.bigNumberify(state.aliceBalance),
+      ethers.utils.bigNumberify(state.bobBalance)
+    ];
+
+    const [myBalance, counterpartyBalance] = [
```
Comment (cf19drofxots) [inline packages/playground/src/components/app-root/app-root.tsx]: Both of these seem to be getting the same balance?

File: packages/playground/src/components/app-root/app-root.tsx [hunk 2]
```diff
@@ -255,26 +257,50 @@ export class AppRoot {
...
+      balances[
+        [state.alice, state.bob].findIndex(
+          address => address === getAddress(nodeAddress, 0)
+        )
+      ],
+      balances[
+        [state.alice, state.bob].findIndex(
+          address => address !== getAddress(nodeAddress, 0)
```
Comment (cf19drofxots) [inline packages/playground/src/components/app-root/app-root.tsx]: This `nodeAddress`  has to be the other party's public identifier.

Comment (joelalejandro) [reply packages/playground/src/components/app-root/app-root.tsx]: Is it necessary? I'm basically saying: `OK, you're *not* my address derived from my xpub, so you _have_ to be the counterparty's address`

File: packages/playground/src/components/app-root/app-root.tsx (hunk 1 above)
Comment (joelalejandro) [reply packages/playground/src/components/app-root/app-root.tsx]: You mean `myBalance = balances[0]` and `counterpartyBalance = balances[0]`?

File: packages/playground/src/components/app-root/app-root.tsx (hunk 2 above)
Comment (cf19drofxots) [reply packages/playground/src/components/app-root/app-root.tsx]: Err didn't catch the `!==`

Yeah I suppose this is fine.

File: packages/playground/src/components/app-root/app-root.tsx (hunk 1 above)
Comment (cf19drofxots) [reply packages/playground/src/components/app-root/app-root.tsx]: Stale comment based on misunderstanding in the other comment

File: packages/playground/src/components/account/account-eth-form/account-eth-form.tsx [hunk 3]
```diff
@@ -11,6 +11,7 @@ export class AccountEthForm {
   @Prop() header: string = "";
   @Prop() button: string = "";
   @Prop() disabled: boolean = false;
+  @Prop() min: number = 0.01;
```
Comment (cf19drofxots) [inline packages/playground/src/components/account/account-eth-form/account-eth-form.tsx]: Is this all that's required to restrict the min amount? I seem to be able to deposit less than this still... :/

Comment (joelalejandro) [reply packages/playground/src/components/account/account-eth-form/account-eth-form.tsx]: Nope, it isn't enough. Forgot something.

Comment (joelalejandro) [reply packages/playground/src/components/account/account-eth-form/account-eth-form.tsx]: This change only covered `/deposit`, not `/exchange`.

Comment (joelalejandro) [reply packages/playground/src/components/account/account-eth-form/account-eth-form.tsx]: Fixed!

**Pull request merged**